app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -242,16 +248,26 @@ class GesturesActivity : AppCompatActivity() {
 
     if (enabled) {
       focalPointLatLng = Point.fromLngLat(-0.12968, 51.50325)
-      // TODO add marker
-      // marker = mapboxMap.addMarker(MarkerOptions().position(focalPointLatLng))
+      pointAnnotationManager =
+        binding.mapView.annotations.createPointAnnotationManager(binding.mapView).apply {
+          BitmapUtils.bitmapFromDrawableRes(
```

Comment:
if it possible that function `fixedFocalPointEnabled` can be called several times with `enabled = true` maybe store  bitmap and reuse it?

Reply:
Added the bitmap to the style when the map is loaded, so no need to regenerate or cache the bitmap in the app.